### PR TITLE
Docker compose and docs refactor

### PR DESCRIPTION
**Pull request by Jnegrier**
Updated docs and added docker-compose file for easy prod grade deployment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,9 @@
...
+# Delta Reporter
 
+Cloud based reporting tool were test results are stored and shown in a friendly way
 
-## Github Pages
+## Documentation Pages
 
 https://delta-reporter.github.io/delta-reporter/
 
-Admin Panel: https://github.com/delta-reporter/delta-reporter/settings (GitHub Pages) section
```
Comment (opishcheiko) [inline README.md:8]: I was thinking to have the details on how to deploy the docs here? Where do you want to put it?

Comment (opishcheiko) [reply README.md:8]: okay, sorry, i saw you moved it to a different folder..
Personally when I am looking for an info in the repo - I look at the main readme.. It's not always obvious that there are other readmes. But maybe in our case it's ok

Comment (opishcheiko) [reply README.md:8]: okay, I understood your idea, forget about the above

**Comment by opishcheiko** [comment]
what's help wanted badge? Do you need help on smthing?

**Comment by Jnegrier** [comment]
> what's help wanted badge? Do you need help on smthing?

Since its documentation, just having a peer review from you its enough help 😸 
I just wanted another view, to see if my approach is correct, thanks!

**Comment by opishcheiko** [comment]
> > what's help wanted badge? Do you need help on smthing?
> 
> Since its documentation, just having a peer review from you its enough help 😸
> I just wanted another view, to see if my approach is correct, thanks!

we can have "ready for review" badge :)

**Pull request merged**